src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java

```diff
@@ -5,6 +5,7 @@
 import net.minecraft.world.entity.EquipmentSlot;
```

Comment:
Please rename MYRIAL_SUIT to MYRIAL_EFFIGY and then add a new ArmorMaterial for the suit; those are not the right defense values for the suit